packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -97,25 +99,10 @@ export function PanelChrome({
   };
 
   const containerStyles: CSSProperties = { width, height };
-
-  const isUsingDeprecatedLeftItems = isEmpty(status) && !loadingState;
-  const showLoading = loadingState === LoadingState.Loading && !isUsingDeprecatedLeftItems;
-  const showStreaming = loadingState === LoadingState.Streaming && !isUsingDeprecatedLeftItems;
-
-  const renderStatus = () => {
-    const showError = loadingState === LoadingState.Error || status?.message;
-    if (!isUsingDeprecatedLeftItems && showError) {
```

Comment:
smilar thing here I don't understand this isUsingDeprecatedLeftItems check, status (statusMessage), is a new prop, if you pass it we use it!